integration/test/upgradetiller/upgrade_tiller_test.go

```diff
@@ -26,11 +26,41 @@ func TestUpgradeTiller(t *testing.T) {
 
 	namespace := "giantswarm"
 	labelSelector := "app=helm,name=tiller"
+	outdatedTillerImage := "gcr.io/kubernetes-helm:v2.7.2"
```

Comment:
In the final PR I'll put this back to our image and `2.8.2 `.

Its worth noting the upgrade will only succeed if the tiller version is different. We have `2.8.1` and `2.8.2` tagged but they have the same tiller version.